versioned_docs/version-1.2/how_to_guides/multiarch.mdx

```diff
@@ -0,0 +1,130 @@
+---
+title: Multiple Architecture Image Support
+sidebar_position: 110
+---
+
+# Multiple Architecture Image Support for ORAS
+
+ORAS can help you manage multiple architecture artifacts such as configurations, executables, and container images.
+This example shows how you can use ORAS to manage executables for different operating systems and architectures.
+
+## Pushing the artifacts
```

Comment:
As there are several features demonstrated in this doc are a part of ORAS v1.3.0, we need to create a new version of documentation before merging this PR. This article is supposed to be v1.3.0.